bin/web-push.js

```diff
@@ -0,0 +1,45 @@
+#! /usr/bin/env node
+const webPush = require('web-push');
```

Comment:
If we use `require('../index.js')`, it would work both when installed locally and globally, no?